src/main/java/Main.java

```diff
@@ -0,0 +1,63 @@
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[1 + (int) (Math.random() * 10)];
+        //массив, содержащий от 1 до 10 фигур
+        for (int i = 0;i < figures.length;i++) {
```

Comment:
```suggestion
        for (int i = 0; i < figures.length; i++) {
```
don't forget about checkstyle

Reply:
fixed